vehicle_node:
  ros__parameters:
    timer_period: 0.1